arkane/molpro.py

```diff
@@ -263,60 +263,85 @@ def loadConformer(self, symmetry=None, spinMultiplicity=0, opticalIsomers=1, sym
 
         # Close file when finished
         f.close()
-        return Conformer(E0=(E0*0.001,"kJ/mol"), modes=modes, spinMultiplicity=spinMultiplicity, opticalIsomers=opticalIsomers)
+        return Conformer(E0=(E0*0.001,"kJ/mol"), modes=modes, spinMultiplicity=spinMultiplicity,
+                         opticalIsomers=opticalIsomers)
 
-    def loadEnergy(self,frequencyScaleFactor=1.):
+    def loadEnergy(self, frequencyScaleFactor=1.):
         """
-        Return the f12 energy in J/mol from a Molpro Logfile of a CCSD(T)-f12 job.
-        This function determines which energy (f12a or f12b) to use based on the basis set,
+        Return either the f12 or MRCI energy in J/mol from a Molpro Logfile.
+        If the MRCI job outputted the MRCI+Davidson energy, the latter is returned.
+        For CCSD(T)-f12, the function determines which energy (f12a or f12b) to use based on the basis set,
         which it will parse out of the Molpro file. For the vtz and dtz basis sets f12a is
```

Comment:
I just noticed that this should be `vdz` and not `dtz`. Might as well change that as part of this PR.
```suggestion
        which it will parse out of the Molpro file. For the vtz and vdz basis sets f12a is
```